### DISCUSSION TITLE
Flashing Sonoff TH16 to Tasmota via FTDI

### DISCUSSION THREAD

**Discussion by bkenobi** [arendst/Tasmota#10622] [Q&A] [answered]
I am trying to flash my Sonoff to Tasmota and have been unsuccessful so far.  I'm currently trying to use Tasmotizer 1.2 with an FTDI set to 3.3v.  This adapter has a switch to toggle between 5v and 3.3v logic.  However, the power is still 5v to the board's vcc pin.  Can someone confirm this is ok?  I tried a resistor based voltage divider circuit but it didn't seem to work.  If 5vdc is ok on vcc, I'll focus elsewhere.  I don't see a voltage regulator on the board, so I could see this as a problem.

If the issue isn't power, I'll move on to the software side.

**Comment by ascillato** [comment, marked as answer]
5V will burn your device. If you have already connected 5V to the ESP8266 chip, sorry, your device is burnt.

Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Please check the [Contributing Guideline and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md)

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Community](https://groups.google.com/d/forum/sonoffusers) for forum.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
